src/components/Card.js

```diff
@@ -40,6 +61,16 @@ class Card {
     return cardElement;
   }
 
+  _renderLikes() {
+    const likesCountElement = this._element.querySelector(".card__likes-count");
+    likesCountElement.textContent = this._likes.length;
+  }
```

Comment:
You should change likes button state here depending on the isLiked state.

```js
isLiked() {
 // return true if user liked the card, otherwise false
}

_renderLikes() {
  //set  likes counter content using this._likes.length

  if (this.isLiked()) {
    // add active class to like button
  } else {
    // remove active class from like button
  }
}
```